source/templates.tex

```diff
@@ -9267,9 +9267,8 @@
 \tcode{f}
 is only declared and not defined at the point of the call.
 The program will be ill-formed unless a specialization for
-\tcode{f<const char*>},
-either implicitly or explicitly generated,
-is present in some translation unit.
+\tcode{f<const char*>}
+is explicitly instantiated in some translation unit.
```

Comment:
Please append a cross-reference to [temp.pre] in case some reader is interested in the normative rule.

Reply:
Done